### ISSUE TITLE
brave rewards: captcha difficulty: win10 x64

### ISSUE THREAD

**Issue by lukemulks** [brave/brave-browser#1350]
## Description 

Experiencing issues with the captcha in Brave Rewards on windows 10 x64. 

I've tested on two separate machines, with different display sizes, resolutions and densities. 

On one machine, I was able to successfully claim my token grant, but it took 9x attempts to solve the captcha in order to successfully claim the grant. 

On the second machine, I was unable to successfully claim the token grant. I made two separate attempts, and was unable to successfully claim the grant on both attempts. 

Each attempt involved ~27x attempts to solve the captcha, with the final attempt resulting in the instructions no longer updating as they had for previous attempts. 

After attempt 27, additional attempts appear to either hang, or fail to register (I'm unable to go further - the instructions do not update as they had for each previous attempt).

Note that I had no issues solving the captcha in `browser-laptop` with the same machines, and the same display settings. 

Info for each machine: 

```
Machine 1: Alienware 13R2
13" display
OS: win10 x64
display: high DPI (OLED)
resolution: 2560x1440 
scale & layout: 125%
(windows recommends 200% with my config)
captcha: 9x before UGP claim was successful.
```
```
Machine 2: Alienware 15R2
15.6" display
OS: win10 x64
display: UHD
resolution: 3840x2160
scale and layout: 225%
(windows recommends 250% with my config)
captcha: 27x, still not successful.
```

## Steps to Reproduce

   1.  Install Brave Beta
   2.  Navigate to Brave Rewards
   3.  Attempt to claim token grant.

## Actual result:

Was able to claim tokens on one machine after 9 attempts to solve the captcha. 

Was unable to claim tokens on the second machine after 27 attempts to solve the captcha. I made a second attempt on this machine, and the outcome was the same. 

## Expected result:
After ~3 attempts to solve the captcha, I should be able to claim a grant (maybe 4-5) 

## Reproduces how often: 

Likely depends on the machine, OS settings, display resolution and screen scale % set in the display properties in the OS settings. 

## Brave version (chrome://version info)

```
Brave | 0.55.10 Chromium: 70.0.3538.22 (Official Build) beta (64-bit)
-- | --
Revision | ac9418ba9c3bd7f6baaffa0b055dfe147e0f8364-refs/branch-heads/3538@{#468}
OS | Windows
JavaScript | V8 7.0.276.15
Flash | (Disabled)
User Agent | Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36
```

### Reproducible on current release:

no

### Website problems only:  
- Does the issue resolve itself when disabling Brave Shields? 
- Is the issue reproducible on the latest version of Chrome? 

### Additional Information

One thing I find myself wondering as I'm attempting to solve the captcha, is if I'm supposed to center the bat icon over the triangle or align at the top or bottom of the target triangle.

The BAT icon is larger than the target triangle, so the overlap has me doing more self-questioning about alignment to the target than if they were the same size and the alignment between icon and target were flush.

cc: @NejcZdovc @davidtemkin

**Comment by kjozwiak** [comment]
Using Win 10 x64 on a VM that's high DPI, I couldn't claim a grant after trying at least 30 different times. I thin we're going to have issues once more with users with high DPI setups.

![captcha](https://user-images.githubusercontent.com/2602313/47088818-5945c880-d1ed-11e8-9ab7-4ab4c99c9709.gif)

@davidtemkin mind giving a try and seeing if you can reproduce on your end? I remember you setup a Win VM that was also a high DPI.

**Comment by NejcZdovc** [comment]
@kjozwiak I can reproduce this one on my VM as well

**Comment by kjozwiak** [comment]
Thanks for double checking @NejcZdovc! Should we take a look at this and see if we need to block? I know that when we first released CAPCHA, we had the same issue with VM's which lead to a lot of user complaints that it didn't work on various high DPI monitors.

Potentially a blocker or at least a candidate for a hotfix right after the release. @NejcZdovc @davidtemkin thoughts?

BTW, it worked on my laptop that's high DPI but after about ~14 attempts. Results:

* Resolution: 1920 x 1080 (Recommended) using 125% scaling (Recommended) - ~14 attempts 
* Resolution: 1920 x 1080 (Recommended) using 100% scaling - ~1 attempt

**Comment by BashedCode** [comment]
I was experiencing the same issue with high DPI, then I lowered the screen resolution to 1280x960 and was able to claim on the first attempt. I had previously attempted approximately 20 times with no success.

**Comment by GeetaSarvadnya** [comment]
1+ https://community.brave.com/t/unable-to-claim-my-free-tokens/35374

**Comment by Jacalz** [comment]
Having the exact same issue on my school laptop 💻Using a touch pad and a normal 15 inch 1080p screen with touch built in, if it matters 😉

**Comment by ChildishGiant** [comment]
Is this hotfix going to be pushed to stable off-schedule?

**Comment by NejcZdovc** [comment]
@ChildishGiant should be out to stable today/tomorrow

**Comment by kjozwiak** [comment]
Looks like the above fix actually made things a lot worse then before. It's almost impossible to accept a grant on all three platforms. Before, some folks on Win were having issues but with this fix, it's affecting the other platforms. I've tried ~about 20 attempts on all three platforms and couldn't solve the CAPTCHA. Example on macOS:

![captchamacos](https://user-images.githubusercontent.com/2602313/47325540-52c2b100-d632-11e8-91d5-d9a230cc0b64.gif)

We need to either revert this or look into another fix that will address the issue.

Platforms used:

* macOS - FAILED (reproduced by @jenn-rhim & @kjozwiak )
* Windows - FAILED (reproduced by @mandar-brave)
* Linux - FAILED (reproduced by @kjozwiak )

Build being used:


Brave | 0.55.20 Chromium: 70.0.3538.67 (Official Build) (64-bit)
-- | --
Revision | 9ab0cfab84ded083718d3a4ff830726efd38869f-refs/branch-heads/3538@{#1002}

**Comment by kjozwiak** [comment]
Issue mentioned in https://github.com/brave/brave-browser/issues/1350#issuecomment-432029760 was due to the server running out of grants. The following issues were created so something similar doesn't happen in the future:

* https://github.com/brave/brave-browser/issues/1793
* https://github.com/brave/brave-browser/issues/1794

**Comment by kjozwiak** [comment]
Went through verification using the following build under `macOS 10.13.6 x64` - **PASSED**

Brave | 0.55.20 Chromium: 70.0.3538.67 (Official Build) (64-bit)
-- | --
Revision | 9ab0cfab84ded083718d3a4ff830726efd38869f-refs/branch-heads/3538@{#1002}
OS | Mac OS X

**Comment by RaulButuc** [comment]
This is still an on-going issue in your latest release ... over a year and a half later ...

**Comment by aleqx** [comment]
Still an issue a year later still. 3+ years and counting ... come on, folks

**Comment by Denny-UD** [comment]
I have the same issue with brave. I tried two many times but it still says "hmm, not quite".  <br>
I used one technic(method) and I get it. <br>

**The method** is: 
**Step 1** - In the windows search bar write "change the resolution of display" and click on it.
**Step 2** - In the display resolutions select a lower resolution than the current resolution.
**Step 3** - Now, try it. Boom you get the rewards. 

**Note - This method worked for me and I hope it worked for you also.** 
Comment if you have any issues.

**Comment by jbergenson** [comment]
I had been experiencing this issue as well and was completely stumped. I can confirm that the method in the post above mine worked flawlessly after changing my resolution from 4k to 2k -- Proved my humanity in one attempt. Thanks!

> I have the same issue with brave. I tried two many times but it still says "hmm, not quite". I used one technic(method) and I get it.
> 
> **The method** is: **Step 1** - In the windows search bar write "change the resolution of display" and click on it. **Step 2** - In the display resolutions select a lower resolution than the current resolution. **Step 3** - Now, try it. Boom you get the rewards.
> 
> **Note - This method worked for me and I hope it worked for you also.** Comment if you have any issues.

**Comment by kjozwiak** [comment]
Hopefully https://github.com/brave/brave-browser/issues/17476 fixes/addresses the above issue. It will be part of `1.31.x` which goes out Tuesday. Once you guys receive `1.31.x`, please let me know if the above issue is still occurring 👍

**Comment by monroemann** [comment]
The issue really should not be 'closed'.  It's still ongoing, and still causing madness for many of us.

**Comment by kjozwiak** [comment]
@monroemann as per https://github.com/brave/brave-browser/issues/1350#issuecomment-945036430, this should have been fixed/addressed via `1.31.x` which was released yesterday. Mind giving `1.31.x` a go and see if it resolved the issues you were experiencing?

**Comment by johngilbert2000** [comment]
Just ran into this same issue, in 2022. 

I disabled all extensions. Tried two different monitors (22" and 14"). Zooming in and out doesn't affect the pop up, but I tried it anyway. 

brave://version info
```
Brave | 1.37.111 Chromium: 100.0.4896.79 (Official Build) (64-bit)
-- | --
Revision | 8fb749dcab8700c24213791969e59deb72fee36f-refs/branch-heads/4896@{#1015}
OS | Linux
JavaScript | V8 10.0.139.9
User Agent | Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/100.0.4896.79 Safari/537.36
```
